### DISCUSSION TITLE
Social media contact information for author in presentation?

### DISCUSSION THREAD

**Discussion by nirvdrum** [quarto-dev/quarto-cli#5334] [Q&A]
I'm just getting started with Quarto, so apologies in advanced if this is covered somewhere. In my industry, it's common to put the author's social media handles on title page. Is there a way to specify a Twitter or Mastodon handle with standard front-matter? Ideally, it'd even have an icon to call out what the service is. If that's not supported out of the box, what's the suggested way for augmenting the title slide with this additional info?

**Comment by mcanouil** [comment]
There is a default title slide which you can use or modify to add contents, see <https://quarto.org/docs/journals/templates.html#revealjs-partials>.

The default uses `title`, `subtitle`, `author`, `institute`, and `date`.

For example, here is a modified title-slide:
![](https://raw.githubusercontent.com/RLille/quarto-revealjs-rlille/main/template.png)
Source: <https://github.com/RLille/quarto-revealjs-rlille>.

In addition, you can use Quarto extension to add some feature such as shortcode to add icon, see <https://github.com/mcanouil/quarto-iconify>.
